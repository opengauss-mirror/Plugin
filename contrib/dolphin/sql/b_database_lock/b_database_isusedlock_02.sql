drop database if exists b_databaselock;
create database b_databaselock dbcompatibility 'b';
\c b_databaselock

select is_used_lock('`');
select is_used_lock('~');
select is_used_lock('!');
select is_used_lock('@');
select is_used_lock('#');
select is_used_lock('$');
select is_used_lock('%');
select is_used_lock('^');
select is_used_lock('&');
select is_used_lock('*');
select is_used_lock('(');
select is_used_lock(')');
select is_used_lock('[');
select is_used_lock(']');
select is_used_lock('{');
select is_used_lock('}');
select is_used_lock('-');
select is_used_lock('_');
select is_used_lock('=');
select is_used_lock('+');
select is_used_lock('\');
select is_used_lock('|');
select is_used_lock(';');
select is_used_lock(':');
select is_used_lock('""""');
select is_used_lock(',');
select is_used_lock('.');
select is_used_lock('/');
select is_used_lock('<');
select is_used_lock('>');
select is_used_lock('?');

select is_used_lock('·');
select is_used_lock('~');
select is_used_lock('！');
select is_used_lock('@');
select is_used_lock('#');
select is_used_lock('￥');
select is_used_lock('%');
select is_used_lock('……');
select is_used_lock('&');
select is_used_lock('*');
select is_used_lock('（');
select is_used_lock('）');
select is_used_lock('-');
select is_used_lock('——');
select is_used_lock('=');
select is_used_lock('+');
select is_used_lock('【');
select is_used_lock('】');
select is_used_lock('{');
select is_used_lock('}');
select is_used_lock('、');
select is_used_lock('|');
select is_used_lock('；');
select is_used_lock('：');
select is_used_lock('‘');
select is_used_lock('“');
select is_used_lock('”');
select is_used_lock('，');
select is_used_lock('《');
select is_used_lock('。');
select is_used_lock('》');
select is_used_lock('/');
select is_used_lock('？');
\c postgres
drop database if exists b_databaselock;
