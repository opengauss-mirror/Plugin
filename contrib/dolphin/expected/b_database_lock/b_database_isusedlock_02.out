drop database if exists b_databaselock;
NOTICE:  database "b_databaselock" does not exist, skipping
create database b_databaselock dbcompatibility 'b';
\c b_databaselock
select is_used_lock('`');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('~');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('!');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('@');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('#');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('$');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('%');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('^');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('&');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('*');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('(');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock(')');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('[');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock(']');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('{');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('}');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('-');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('_');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('=');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('+');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('\');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('|');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock(';');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock(':');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('""""');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock(',');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('.');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('/');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('<');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('>');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('?');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('·');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('~');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('！');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('@');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('#');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('￥');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('%');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('……');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('&');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('*');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('（');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('）');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('-');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('——');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('=');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('+');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('【');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('】');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('{');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('}');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('、');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('|');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('；');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('：');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('‘');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('“');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('”');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('，');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('《');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('。');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('》');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('/');
 is_used_lock 
--------------
             
(1 row)

select is_used_lock('？');
 is_used_lock 
--------------
             
(1 row)

\c postgres
drop database if exists b_databaselock;
